### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by mueller-ma**

### CODE REVIEW

File: app/src/main/res/resource/values-de/strings.xml [hunk 1]
```diff
@@ -26,17 +26,17 @@
     <string name="search">SUCHE</string>
     <string name="follow">FOLGEN</string>
     <string name="following">FOLGT</string>
-    <string name="my_follow">Meine Folgungen</string>
+    <string name="my_follow">Anhänger</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Oh no, please not "Anhänger". These are maybe religious or fans of a band, but in social media you just use "Follower" as in English. I've never heard a better word. Also don't forget the "my".
--> "Meine Follower"

File: app/src/main/res/resource/values-de/strings.xml [hunk 2]
```diff
@@ -26,17 +26,17 @@
...
 
-    <string name="liked">Gemocht</string>
+    <string name="liked">Favoriten</string>
     <string name="collected">Gesammelt</string>
-    <string name="followed">Folgende</string>
-    <string name="released">Entlassen</string>
+    <string name="followed">Gefolgt</string>
+    <string name="released">Veröffentlicht</string>
     <string name="published">Publiziert</string>
     <string name="curated">Curated</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: What about "Kuratiert" (as strange as it sounds…). It's the literal translation: https://dict.leo.org/englisch-deutsch/Curated https://de.wiktionary.org/wiki/kuratieren

A more common one would maybe be "Gesammelt"… Depends on how this is used.
**Edit**: Oh, just see it is already used for "collected", so we need to know the difference between these two strings.

File: app/src/main/res/resource/values-de/strings.xml [hunk 3]
```diff
@@ -56,20 +56,22 @@
     <string name="unsplash">Unsplash.com</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:56]: I cannot comment line 46, but why "Sie". Such formal? Why not just "du" und "deine"…
We're on the internet here, so let's don't be such formal. :wink:

Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:56]: l. 50: LOL, "relative"? What about "ähnliche Fotos"? (or one of the [thousand other translations](https://dict.leo.org/englisch-deutsch/related)) Same for "Sammlungen" in l. 51.
In any case "relative" is wrong here.

File: app/src/main/res/resource/values-de/strings.xml [hunk 4]
```diff
@@ -56,20 +56,22 @@
...
     <string name="about_unsplash">Freie (mach, was immer du willst) hochauflösende Fotos.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:57]: Mhh, @WangDaYeeeeee, does "free" in this slogan really refer to freedom or to gratis ("free as in beer")? I mean the pictures are obviously both, but for a slogan maybe "kostenlos" or "gratis" sound better…

File: app/src/main/res/resource/values-de/strings.xml [hunk 5]
```diff
@@ -56,20 +56,22 @@
...
     <string name="retrofit">Retrofit 2.0</string>
-    <string name="about_retrofit">Type-safe HTTP client für Android und Java von Square, Inc.</string>
+    <string name="about_retrofit">Type-safe HTTP-Client für Android und Java von Square, Inc.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Come on: "Typensicherer HTTP-Client […]"

File: app/src/main/res/resource/values-de/strings.xml [hunk 6]
```diff
@@ -56,20 +56,22 @@
...
-    <string name="about_photo_view">Ein Widget zur Vorschau und Skalierung des Bildes</string>
+    <string name="photo_view">PhotoView</string>
+    <string name="about_photo_view">Ein Widget zur Vorschau und Skalierung von Bildern</string>
     <string name="page_indicator">InkPageIndicator</string>
     <string name="about_page_indicator">InkPageIndicator erstellt von @nickbutcher für Plaid https://github.com/nickbutcher/plaid und gebackportet von David Pacioianu für API 14+ (4.0+).</string>
     <string name="greendao_db">greenDAO</string>
     <string name="about_greendao_db">greenDAO ist eine leichte und schnelle Lösung für Android, die Objekte zu SQLite Datenbanken abbildet.</string>
-
+    <string name="butter_knife">Butter Knife</string>
+    <string name="about_butter_knife">Verbinde Android views und callbacks mit Feldern und Methoden.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Please write views and callbacks with capital letters in German. Also [durchkoppeln](https://de.wikipedia.org/wiki/Durchkopplung) bitte…

"Android-Views"

File: app/src/main/res/resource/values-de/strings.xml [hunk 7]
```diff
@@ -102,7 +104,7 @@
     <string name="feedback_create_file_failed">Dateierstellung fehlgeschlagen.</string>
     <string name="feedback_need_permission">Berechtigung wurde nicht erteilt.</string>
     <string name="feedback_choose_share_app">Foto teilen</string>
-    <string name="feedback_choose_wallpaper_app">Hitergrundbild setzen</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:105]: Haha, what did you have in mind, when translating this? :stuck_out_tongue_winking_eye: 

Now, it's okay, of course.

File: app/src/main/res/resource/values-de/strings.xml [hunk 8]
```diff
@@ -112,12 +114,12 @@
     <string name="feedback_exposure">Belichtungszeit</string>
     <string name="feedback_aperture">Blende</string>
     <string name="feedback_focal">Brennweite</string>
-    <string name="feedback_iso">Iso</string>
+    <string name="feedback_iso">ISO</string>
 
     <string name="feedback_portfolio_is_null">Portfolio Link ist ungültig.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Durchkoppeln? Okay, no, I'd rather write it as one word: "Portfoliolink". Or, when you don't like that, "Link zum Portfolio".

File: app/src/main/res/resource/values-de/strings.xml [hunk 9]
```diff
@@ -112,12 +114,12 @@
...
     <string name="feedback_no_filter">Kein Filterelement.</string>
 
-    <string name="feedback_request_token_failed">Zugriffsfaktoranfrage gescheitert.</string>
+    <string name="feedback_request_token_failed">Anfrage für Zugriffstoken gescheitert.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Technically I'd rather say in this case it is a "Abfrage des […]tokens".

File: app/src/main/res/resource/values-de/strings.xml [hunk 10]
```diff
@@ -112,12 +114,12 @@
...
     <string name="feedback_login_text">Mit Ihrem Unsplash Account einloggen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: so formal…

File: app/src/main/res/resource/values-de/strings.xml [hunk 11]
```diff
@@ -112,12 +114,12 @@
...
     <string name="feedback_please_login">Bite melden Sie sich an.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: also formal.

File: app/src/main/res/resource/values-de/strings.xml [hunk 12]
```diff
@@ -142,7 +144,7 @@
     <string name="feedback_name_is_required">Name ist nötig</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:144]: E-Mail-Adresse also durchkoppeln. line: https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR130

Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:144]: Maybe "erforderlich". Sounds better…

File: app/src/main/res/resource/values-de/strings.xml [hunk 13]
```diff
@@ -142,7 +144,7 @@
...
     <string name="feedback_sure">Sicher?</string>
 
-    <string name="feedback_notify_set_back_to_top">Sie können den Rücktasteneffekt einstelleln</string>
+    <string name="feedback_notify_set_back_to_top">Sie das Verhalten der "Zurück" Taste ändern</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Now this is no sentence anymore. Note that the English original even has a dot `.` at the end, so you should make this a sentence here and also include the trailing dot.

File: app/src/main/res/resource/values-de/strings.xml [hunk 14]
```diff
@@ -154,30 +156,30 @@
     <string name="feedback_exceed_rate_limit">Ups! Es gab zu viele Besucher in dieser Stunde, bitte versuchen Sie in der nächsten Stunde erneut.</string>
 
     <string name="feedback_confirm_change_theme">Themaänderung bestätigen</string>
-    <string name="feedback_change_theme_warning">Jetztiges Ändern des Themas wird aktuelle downloads abbrechen.</string>
+    <string name="feedback_change_theme_warning">Änderung des Themas wird alle aktuelle Downloads abbrechen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: No (nice) sentence. At least begin it with "Eine" or "Diese"

File: app/src/main/res/resource/values-de/strings.xml [hunk 15]
```diff
@@ -154,30 +156,30 @@
...
     <string name="feedback_confirm_restart">Neustart bestätigen</string>
-    <string name="feedback_restart_warning">Jetztiges Neustarten der App wird alle Downloads abbrechen.</string>
+    <string name="feedback_restart_warning">Neustarten der App wird alle aktuellen Downloads abbrechen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Here too. "Ein"/"Der Neustart" or "Das Neustarten […]"

File: app/src/main/res/resource/values-de/strings.xml [hunk 16]
```diff
@@ -154,30 +156,30 @@
...
 
     <string name="feedback_click_again_to_exit">Zum Verlassen nochmal drücken.</string>
 
     <string name="feedback_share_photo_title">Ein Foto von Unsplash.com</string>
-    <string name="feedback_share_photo_extra">Genommen von # am #</string>
+    <string name="feedback_share_photo_extra">Genommen von # am #\n</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: When the first # is the author and the second the date, this sounds strange. "Genommen von H.P.X am 17.05.2017"
Maybe use "Aufgenommen", which could be the correct translation as "the photo was taken by XY at Z".

File: app/src/main/res/resource/values-de/strings.xml [hunk 17]
```diff
@@ -154,30 +156,30 @@
...
-    <string name="feedback_share_collection_extra">Erstellt von # am #</string>
+    <string name="feedback_share_collection_extra">Erstellt von # am #\n</string>
     <string name="feedback_share_user_title">Ein Benutzerprofil von Unsplash.com</string>
-    <string name="feedback_share_user_extra">#</string>
+    <string name="feedback_share_user_extra">#\n</string>
 
-    <string name="feedback_like_failed">Liken des Fotos fehlgeschlagen.</string>
-    <string name="feedback_unlike_failed">Unliken des Fotos fehlgeschlagen.</string>
+    <string name="feedback_like_failed">Favorisieren des Fotos fehlgeschlagen.</string>
+    <string name="feedback_unlike_failed">Als Favorit entfernen fehlgeschlagen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:173]: Sounds strange, but difficult to translate. Maybe "Favorisierung entfernen", "Favorit löschen", "Entfernen des Favoriten[statuses]"…

File: app/src/main/res/resource/values-de/strings.xml [hunk 18]
```diff
@@ -154,30 +156,30 @@
...
 
     <string name="feedback_follow_failed">Folgen fehlgeschlagen.</string>
-    <string name="feedback_cancel_follow_failed">Unfolgen fehlgeschlagen.</string>
+    <string name="feedback_cancel_follow_failed">Nicht mehr folgen ist fehlgeschlagen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: I think "Entfolgen" is okay here, this sounds very strange as you don't know that the first 4 words are a single action. Alternatively put it in brackets: `"Nicht mehr folgen" ist fehlgeschlagen."

File: app/src/main/res/resource/values-de/strings.xml [hunk 19]
```diff
@@ -154,30 +156,30 @@
...
 
     <string name="feedback_story_is_null">Geschichte existiert nicht oder wurde noch nicht geladen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: I'm not entirely sure whether a "feedback story" is a "Geschichte" here…

File: app/src/main/res/resource/values-de/strings.xml [hunk 20]
```diff
@@ -154,30 +156,30 @@
...
 
     <!-- introduce. -->
     <string name="introduce_title_back_top">Nach oben scrollen</string>
-    <string name="introduce_description_back_top">Tippen Sie auf die TopBar uder Zurück, die Fotolistenansicht wird zurück nach oben scrollen.</string>
+    <string name="introduce_description_back_top">Tippen Sie auf die TopBar oder Zurück, um in der Listenansicht nach oben zu scrollen.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: You can translate TopBar. Just "obere Leiste".
Also "oder auf Zurück" or, better, "oder auf den Zurück-Knopf".

File: app/src/main/res/resource/values-de/strings.xml [hunk 21]
```diff
@@ -154,30 +156,30 @@
...
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Missing: `<string name="muzei_description">Featured Photos from Unsplash</string>`

"Empfohlene Fotos von Unsplash" (or when it is meant like that: "Von Unsplash empfohlene/vorgeschlagene Fotos"

File: app/src/main/res/resource/values-de/strings.xml [hunk 22]
```diff
@@ -190,7 +192,7 @@
     <string name="action_about">Über</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml:192]: https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR186
"Following" is likely a button/list header here. "Folgende" is clearly wrong. Just use the English word: "Followers". Or "Deine Followers"?

File: app/src/main/res/resource/values-de/strings.xml [hunk 23]
```diff
@@ -223,21 +225,21 @@
     <string name="action_align_right">Rechts anheften</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: "Portfolio Link öffneb" Typo & durchkoppeln/ein Wort (wie vorhin, weiter oben).

https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR211

Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR212
"einreichen"? Maybe "übermitteln" is better as this is not that formal?

Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR217 again durchkoppeln. Maybe also call it "Download-URL".

Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: This "anheften" stuff is usually translated as "ausrichten". Just look in your favourite word processing program.
Also "Zentriert ausrichten".

File: app/src/main/res/resource/values-de/strings.xml [hunk 24]
```diff
@@ -223,21 +225,21 @@
...
 
     <string name="action_wallpaper">Hintergrundbild</string>
-    <string name="action_lockscreen">LockScreen</string>
+    <string name="action_lockscreen">Sperrbildschirm</string>
     <string name="action_wallpaper_lockscreen">Hintergrundbild und Sperrbildschirm</string>
 
     <!-- settings. -->
-    <string name="settings_category_basic">Basisch</string>
-    <string name="settings_title_back_to_top">Drücken Sie den Zurückknopf um nach oben zurückzuscrollen</string>
+    <string name="settings_category_basic">Haupteinstellungen</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Why not just "Grundlagen"? Although "Haupteinstellungen" is also not bad.

File: app/src/main/res/resource/values-de/strings.xml [hunk 25]
```diff
@@ -223,21 +225,21 @@
...
+    <string name="settings_title_back_to_top">Drücken Sie den "Zurück" Knopf um nach oben zurückzuscrollen</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: No, previously it was better. "Zurück-Knopf" (no brackets necessary)

File: app/src/main/res/resource/values-de/strings.xml [hunk 26]
```diff
@@ -223,21 +225,21 @@
...
     <string name="settings_title_custom_api_key">Eigener API Schlüssel</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Durchkoppeln!

File: app/src/main/res/resource/values-de/strings.xml [hunk 27]
```diff
@@ -223,21 +225,21 @@
...
     <string name="settings_summary_custom_api_key">Legen Sie Ihren eigenen API Key und Geheimnis fest.</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Now you won't translate "API key"? So "API-Schlüssel" too here.

File: app/src/main/res/resource/values-de/strings.xml [hunk 28]
```diff
@@ -223,21 +225,21 @@
...
-    <string name="settings_title_default_collection_type">Standartkollektionstyp</string>
+    <string name="settings_title_default_order">Standardordnung</string>
+    <string name="settings_title_default_collection_type">Standardkollektionstyp</string>
 
     <string name="settings_category_download">Herunterladen</string>
     <string name="settings_title_download_scale">Skalierung zum Herunterladen</string>
 
-</resources>
\ No newline at end of file
+</resources>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: Many things missing:

https://github.com/WangDaYeeeeee/Mysplash/blob/master/app/src/main/res/resource/values/strings.xml#L288-L306

File: app/src/main/res/resource/values-de/strings.xml [hunk 29]
```diff
@@ -223,21 +225,21 @@
...
     <string name="settings_title_language">Sprache</string>
 
     <string name="settings_category_filter">Filter</string>
-    <string name="settings_title_default_order">Standartordnung</string>
-    <string name="settings_title_default_collection_type">Standartkollektionstyp</string>
+    <string name="settings_title_default_order">Standardordnung</string>
+    <string name="settings_title_default_collection_type">Standardkollektionstyp</string>
 
     <string name="settings_category_download">Herunterladen</string>
     <string name="settings_title_download_scale">Skalierung zum Herunterladen</string>
```
Comment (rugk) [inline app/src/main/res/resource/values-de/strings.xml]: I doubt you can download a scale here. Funnily scale can also mean "Waage" or so…
Maybe "Skaliert herunterladen" (= "in einer bestimmten Skalierung")?

Or, if there are multiple options connected with this option: "Skalierung beim Herunterladen"

**Review by rugk — CHANGES_REQUESTED** [review]
still something necessary, also please check for further missing strings, I only noticed some here.

**Comment by rugk** [comment]
BTW you can also invite @OffifialMITX as a collaborator to your repo (with the PR) and then one can edit it online, at least.

File: app/src/main/res/resource/values-de/strings.xml (hunk 4 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml:57]: Gratis and kostenlos is only free like in beer, not freedom

File: app/src/main/res/resource/values-de/strings.xml (hunk 7 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml:105]: ;)

File: app/src/main/res/resource/values-de/strings.xml (hunk 8 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml]: Maybe there is a better word than portfolio

File: app/src/main/res/resource/values-de/strings.xml (hunk 10 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml]: deinem oder Deinem?

File: app/src/main/res/resource/values-de/strings.xml (hunk 12 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml:144]: Dein or Ein Name?

File: app/src/main/res/resource/values-de/strings.xml (hunk 16 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml]: ok, i thought its something like: von unsplash genommen

File: app/src/main/res/resource/values-de/strings.xml (hunk 25 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml]: Zurück-Knopf

**Comment by mueller-ma** [comment]
@offifialmitx Can send me a PR if needed

File: app/src/main/res/resource/values-de/strings.xml (hunk 4 above)
Comment (rugk) [reply app/src/main/res/resource/values-de/strings.xml:57]: Yes, that's why I am asking this. I don't know what was intended/meant when writing the slogan…

File: app/src/main/res/resource/values-de/strings.xml (hunk 12 above)
Comment (rugk) [reply app/src/main/res/resource/values-de/strings.xml:144]: Think "Ein" is okay.

**Comment by rugk** [comment]
A PR for a PR… well… yes, possible. :smile:

File: app/src/main/res/resource/values-de/strings.xml (hunk 4 above)
Comment (mueller-ma) [reply app/src/main/res/resource/values-de/strings.xml:57]: https://unsplash.com/license
Freedom

**Pull request merged**